script/lib/generate-metadata.js

```diff
@@ -27,8 +27,8 @@ function buildBundledPackagesMetadata () {
     const packagePath = path.join(CONFIG.intermediateAppPath, 'node_modules', packageName)
     const packageMetadataPath = path.join(packagePath, 'package.json')
     const packageMetadata = JSON.parse(fs.readFileSync(packageMetadataPath, 'utf8'))
-    normalizePackageData(packageMetadata, () => {
-      throw new Error(`Invalid package metadata. ${metadata.name}: ${msg}`)
+    normalizePackageData(packageMetadata, (msg) => {
+      throw new Error(`Invalid package metadata. ${packageName}: ${msg}`)
```

Comment:
`metadata.name` doesn't exist, but `packageName` does.  I'm not sure the two are equivalent without knowing what `metadata.name` was supposed to refer to.